server/resources.py

```diff
@@ -46,11 +45,11 @@ def post(self):
             return {'message': 'User {} already exists'.format(data['email'])}, 409
 
         new_user = UserModel(
-            email = data['email'],
-            credentials = data['credentials']
```

Comment:
Personally I don't think credentials should be added during registration. The request body from the front end won't have a credentials to pass because the google authorization doesn't happen until after. Don't mind merging tho just to get everyone on the same page first cause it seems like it'll be good as far as allowing everyone to get on the same page

Reply:
Agreed, I was thinking the same thing. It was required since it was added to the database so I've left it in for now. We can always remove